src/bin/fafnir.rs

```diff
@@ -0,0 +1,35 @@
+extern crate fafnir;
+extern crate postgres;
+#[macro_use]
+extern crate structopt;
+extern crate mimir;
+extern crate mimirsbrunn;
+
+#[derive(StructOpt, Debug)]
+struct Args {
+    /// Postgresql parameters
+    #[structopt(long = "pg")]
+    pg: String,
+    /// Elasticsearch parameters.
+    #[structopt(long = "connection-string", default_value = "http://localhost:9200/")]
+    connection_string: String,
```

Comment:
completly unrelated to this PR, but we should change this parameter since it's also a `connection-string` for `pg` and rename it to `es`

Reply:
done.